### PR TITLE
holochain_net: configure log level

### PR DESCRIPTION
**Pull request by ddd-mtl**
- [x] I have added a summary of my changes to the changelog

### HC_N3H_LOG_LEVEL
Set the logging level used globally by N3H. Must be one of the following: 't', 'd', 'i', 'w', 'e'

#### Misc

Made tweetlogging macros available within the 'net' crate.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 ### Changed
+- New network setting N3H_LOG_LEVEL
```
Comment (Connoropolous) [inline CHANGELOG.md]:
```suggestion
- New network setting via environment variable HC_N3H_LOG_LEVEL
```

Comment (Connoropolous) [reply CHANGELOG.md]: So let me get this straight, N3H_LOG_LEVEL is actually for n3h, then there is a SEPARATE env var for HC that controls that ENV var?

**Comment by Connoropolous** [comment]
So let me get this straight, N3H_LOG_LEVEL is actually for n3h, then there is a SEPARATE env var for HC that controls that ENV var?

**Comment by ddd-mtl** [comment]
@Connoropolous Yes, and Done :)

**Pull request merged**